src/ucan-cloud/store/state/index.ts

```diff
@@ -2,10 +2,10 @@ import { runtimeFn } from "@adviser/cement";
 
 export default async function store(name: string) {
   if (runtimeFn().isNodeIsh) {
-    const n = await import("./node.js");
+    const n = await import("../../../ucan-cloud/store/state/node.js");
     return n.default(name);
   } else {
-    const b = await import("./browser.js");
+    const b = await import("../../../ucan-cloud/store/state/node.js");
```

Comment:
Sorry, my mistake, this path should end with `browser.js`

Reply:
Oops, good catch.